### PR TITLE
[Image] Rewrite for V3

### PR DESCRIPTION
**Pull request by palmtree5**
This PR provides a rewrite of the Image cog. In addition to the minor changes, this migrates the imgur-related commands from imgurpython (and thus requests) to aiohttp. `[p]imgur random` has also been removed as the part of it with no args (getting a random set of gallery images) is deprecated according to Imgur's API docs and the part with a term may as well have been invoking `[p]imgur search`

### CODE REVIEW

**Comment by tekulvw** [comment]
@palmtree5 are you done developing this/is it ready for testing?

**Review by Twentysix26 — DISMISSED** [review]
This needs to be converted from JsonDB to config

**Comment by palmtree5** [comment]
@tekulvw not ready given it's still on JsonDB

**Comment by palmtree5** [comment]
This is ready now that I've switched it to `Config`

**Comment by atiwiex** [comment]
Real config*

File: cogs/image/image.py [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+
+class Image:
+    """Image related commands."""
+    default_global = {
+        "imgur_client_id": None
+    }
+
+    def __init__(self, bot):
+        self.bot = bot
+        self.settings = Config.get_conf(self, identifier=2652104208, force_registration=True)
```
Comment (tekulvw) [inline cogs/image/image.py:18]: Didn't register defaults

**Review by tekulvw — CHANGES_REQUESTED** [review]

**Review by tekulvw — APPROVED** [review]

**Pull request merged**